<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FilterSet</class>
 <widget class="QDialog" name="FilterSet">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>365</width>
    <height>180</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Установить фильтр</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../resources.qrc">
    <normaloff>:/plugins/openland/icons/filter.png</normaloff>:/plugins/openland/icons/filter.png</iconset>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QRadioButton" name="radioButtonNone">
     <property name="text">
      <string>Убрать фильтр по кварталам/участкам</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QRadioButton" name="radioButtonKvr">
     <property name="text">
      <string>По выбранному кварталу(кварталам)</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QRadioButton" name="radioButtonUch">
     <property name="text">
      <string>По выбранному участку(участкам)</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="checkBoxOnlyCurrentUser">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Только объекты пользователя БД из настроек</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>radioButtonNone</tabstop>
  <tabstop>radioButtonKvr</tabstop>
  <tabstop>radioButtonUch</tabstop>
  <tabstop>checkBoxOnlyCurrentUser</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources>
  <include location="../resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>FilterSet</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>FilterSet</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
